bcda/client/bluebutton.go

```diff
@@ -256,3 +259,13 @@ func HashIdentifier(toHash string) (hashedValue string) {
 	}
 	return hex.EncodeToString(pbkdf2.Key([]byte(toHash), pepper, blueButtonIter, 32, sha256.New))
 }
+
+func UpdateParamWithLastUpdated(params *url.Values, since string, jobCreation time.Time) {
+	// only set the lower bound parameter if it exists and begins with "ge" (to align with what is expected in _lastUpdated)
+	if len(since) > 0 && strings.HasPrefix(since, "ge") {
+		params.Set("_lastUpdated", since)
+
+		// now add the upper bound based on job creation
+		params.Add("_lastUpdated", "lt"+jobCreation.Format(time.RFC3339Nano))
+	}
```

Comment:
Does the BFD accept a nanosecond precision value? FHIR is a millisecond precision specification. HAPI which is the BFD validating library for FHIR is pretty particular about this. I've tried other precisions and found that it is not accepted.

Reply:
BFD is accepting our nanosecond requests.  This also appears to align with the FHIR `dateTime` format:
https://www.hl7.org/fhir/datatypes.html#dateTime

Relevant portion of the regex is `(\.[0-9]+)`